lib/active_admin/views/pages/index.rb

```diff
@@ -130,7 +130,8 @@ def render_index
           per_page         = config.fetch(:per_page, active_admin_config.per_page)
 
           paginated_collection(collection, entry_name:       active_admin_config.resource_label,
-                                           entries_name:     active_admin_config.plural_resource_label(count: collection_size),
+                                           entries_name:     active_admin_config.plural_resource_label(
+                                                               count: collection_size(collection)),
```

Comment:
Line is too long. [99/80]<br>Indent the first parameter one step more than the previous line.